### ISSUE TITLE
NPM install?

### ISSUE THREAD

**Issue by haysclark** [lordpengwin/muzak#1]
Curious if the README should just suggest that people use 'npm'? e.g.:

```
npm install https://github.com/lordpengwin/squeezenode
```

**Comment by heylookltsme** [comment]
Agree. Also the muzak repo should include a `package.json` file that lists all the npm dependencies (which I guess is just squeezenode) so folks can simply run `npm install`.

**Comment by heylookltsme** [comment]
I have a PR to add a package.json file here: https://github.com/MikeDeSantis/muzak/pull/2
